app/gui/src/project-view/components/GraphEditor/widgets/WidgetTableEditor/TableHeader.vue

```diff
@@ -98,12 +99,12 @@ function onMouseRightClick(event: MouseEvent) {
 
 <template>
   <SvgButton
-    v-if="params.type === 'newColumn'"
+    v-if="columnParams.type === 'newColumn'"
```

Comment:
It seems like more of this component is dependent on `columnParams.type` than not; would it make sense to split it into separate components?

Reply:
I will leave it now; technically I could define a separate component for every column, but this is not easy after all refactoring & wrappers.